### PR TITLE
forbid bcmath to return negative zero numbers

### PR DESCRIPTION
**Pull request by drdzyk**
bcmath currently able to return numbers like -0, -0.0 etc. This behavior was fixed in php 7.4.23, 8.0.10, see https://bugs.php.net/bug.php?id=78238 for more details.

### CODE REVIEW

**Review by DrDet — CHANGES_REQUESTED** [review]
I think it would be much better to keep it in phpt tests, and not to mix it up with end-to-end tests for HTTP server.
Let's do it in 483_bcmath.php. For those tests which return 0 or -0 in different php versions it will look like this:
```
  #ifndef KPHP
  if (version_compare(PHP_VERSION, "7.4.23") >= 0) {
  #endif
    bcadd("0", "-0.1"); // 0
    bcmul("0.999999999", "-0.111111111111"); // 0
  #ifndef KPHP
  } else {
    echo "0\n";
    echo "0\n";
  }
  #endif
```

**Comment by drdzyk** [comment]
> I think it would be much better to keep it in phpt tests, and not to mix it up with end-to-end tests for HTTP server. Let's do it in 483_bcmath.php. For those tests which return 0 or -0 in different php versions it will look like this:
> 
> ```
>   #ifndef KPHP
>   if (version_compare(PHP_VERSION, "7.4.23") >= 0) {
>   #endif
>     bcadd("0", "-0.1"); // 0
>     bcmul("0.999999999", "-0.111111111111"); // 0
>   #ifndef KPHP
>   } else {
>     echo "0\n";
>     echo "0\n";
>   }
>   #endif
> ```

Cool! fixed

File: tests/phpt/dl/483_bcmath.php [hunk 1]
```diff
@@ -121,3 +121,43 @@
...
+  In new php versions and in kphp bcmath never returns numbers like -0, -0.0 (but returns 0, 0.0).
+  See https://bugs.php.net/bug.php?id=78238 for more details.
+*/
+  #ifndef KPHP
+  if ((version_compare(PHP_VERSION, "7.4.23") >= 0) &&
+       (version_compare(PHP_VERSION, "8.0.0") < 0) ||
+       (version_compare(PHP_VERSION, "8.0.10") >= 0)) {
+  #endif
+    var_dump(bcadd("0", "-0.1"));    // 0
+    var_dump(bcadd("0", "-0.1", 1)); // -0.1
```
Comment (DrDet) [inline tests/phpt/dl/483_bcmath.php:136]: Why do we need to keep tests returning non-zero here? If I'm not mistaken, their behavior must be the same in different php versions and kphp

Comment (drdzyk) [reply tests/phpt/dl/483_bcmath.php:136]: In order to test that code near is not broken.

**Review by DrDet — APPROVED** [review]
